app/models/manageiq/providers/embedded_ansible/automation_manager/event_catcher/runner.rb

```diff
@@ -1,3 +1,12 @@
 class ManageIQ::Providers::EmbeddedAnsible::AutomationManager::EventCatcher::Runner < ManageIQ::Providers::BaseManager::EventCatcher::Runner
   include ManageIQ::Providers::AnsibleTower::Shared::AutomationManager::EventCatcher::Runner
+
+  def start_event_monitor
+    tid = super
+    return tid unless tid.nil?
+
+    # Get a new copy of the ems record in case the embedded ansible role changed servers
+    @ems.reload
```

Comment:
so I am not very familiar with the life cycle, 3 (maybe obvious to others) questions,
* Do we need to reload before the call to `super` so that whatever being done in `super` will reflect the new state in ems?
* The `url` is stored in endpoint which will be reloaded as well, right?
* Is this behavior applicable to other managers?